docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -14,24 +14,65 @@ connections to a remote cluster. This functionality is used in
 endif::[]
 
 Remote cluster connections work by configuring a remote cluster and connecting
-only to a limited number of nodes in the remote cluster. Each remote cluster is
-referenced by a name and a list of seed nodes. When a remote cluster is
-registered, its cluster state is retrieved from one of the seed nodes so that by
-default up to three _gateway nodes_ are selected to be connected to as part of
-remote cluster requests.  Remote cluster connections consist of uni-directional
-connections from the coordinating node to the previously selected remote nodes
-only. You can tag which nodes should be selected by using node attributes (see <<remote-cluster-settings>>).
+only to a limited number of nodes in that remote cluster. Each remote cluster
+is referenced by a name and a list of seed nodes. When a remote cluster is
+registered, its cluster state is retrieved from one of the seed nodes and up
+to three _gateway nodes_ are selected to be connected to as part of remote
+cluster requests. All the communication required between different clusters
```

Comment:
> three _gateway nodes_ are selected to be connected to as part of remote cluster requests

It is unclear at this point in the page whether those three gateway nodes are used permanently or just for a particular request or whether the selection can change over time...?

Reply:
They are used for all CCR and CCS requests from then on. The selection may change if one or more of those nodes are shut down, in that case they will get replaced.

Reply:
Thanks, I think we should add that piece of info here.